<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgOpportunities</class>
 <widget class="QWidget" name="wdgOpportunities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/bank.png</normaloff>:/xulpymoney/bank.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Opportunities list</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Opportunity annotations list</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="wdgYear" name="wdgYear" native="true"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="cmbMode">
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <item>
                <property name="text">
                 <string>Show current opportunities</string>
                </property>
                <property name="icon">
                 <iconset resource="../images/xulpymoney.qrc">
                  <normaloff>:/xulpymoney/new.png</normaloff>:/xulpymoney/new.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Show removed opportunities</string>
                </property>
                <property name="icon">
                 <iconset resource="../images/xulpymoney.qrc">
                  <normaloff>:/xulpymoney/expired.jpg</normaloff>:/xulpymoney/expired.jpg</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Show executed opportunities</string>
                </property>
                <property name="icon">
                 <iconset resource="../images/xulpymoney.qrc">
                  <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Show all</string>
                </property>
                <property name="icon">
                 <iconset resource="../images/xulpymoney.qrc">
                  <normaloff>:/xulpymoney/eye.png</normaloff>:/xulpymoney/eye.png</iconset>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="mqtwDataWithObjects" name="mqtwOpportunities" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Amount to invest in opportunity product currency</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQLineEdit" name="txtInvest">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOpportunityNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New purchase opportunity</string>
   </property>
   <property name="toolTip">
    <string>New purchase opportunity</string>
   </property>
  </action>
  <action name="actionOpportunityDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/button_cancel.png</normaloff>:/xulpymoney/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Delete purchase opportunity</string>
   </property>
   <property name="toolTip">
    <string>Delete purchase opportunity</string>
   </property>
  </action>
  <action name="actionOpportunityEdit">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Edit purchase opportunity</string>
   </property>
   <property name="toolTip">
    <string>Edit purchase opportunity</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
   </property>
   <property name="text">
    <string>Execute opportunity</string>
   </property>
   <property name="toolTip">
    <string>Execute opportunity</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/expired.jpg</normaloff>:/xulpymoney/expired.jpg</iconset>
   </property>
   <property name="text">
    <string>Remove opportunity</string>
   </property>
   <property name="toolTip">
    <string>Remove opportunity</string>
   </property>
  </action>
  <action name="actionShowGraphic">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/curiosity.png</normaloff>:/xulpymoney/curiosity.png</iconset>
   </property>
   <property name="text">
    <string>Show opportunity graphic</string>
   </property>
   <property name="toolTip">
    <string>Show opportunity graphic</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgYear</class>
   <extends>QWidget</extends>
   <header location="global">wdgYear</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
  <customwidget>
   <class>mqtwDataWithObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
